Update patterns-01-fr.json

pcwsmith opened the pull request:
Correcting the record for the Date de modification pattern so that it appears in the table on the Bibliothèque de modèles et de configurations de conception. If I'm not mistaken, it just needs TANDP=1 rather than  0. No change needed in the equivalent english file, as it is already correct.

pcwsmith commented:
@delisma or @prycrane I'm not able to review this pull request the way I want. The only thing this PR does is change a value in the file https://github.com/canada-ca/systeme-conception/blob/master/ajax/patterns-01-fr.json such that the record for one of the items in the file gets included in the table on the Bibliothèque de modèles et configurations ...

I appended /test to the Netlify preview link, which takes me to  https://deploy-preview-149--systeme-conception-canada-ca.netlify.app/test, a version of the FR d.s. landing page, but the page I need to review is the FR template and pattern library. The link to the FR library from  https://deploy-preview-149--systeme-conception-canada-ca.netlify.app/test takes me to the FR library on the production site. So I can't check to see if the change made in this PR works. Help?

delisma commented:
On the test page when you filter the table to show only the templates, does your changes get reflected?

pcwsmith commented:
Yep it was already included in the table on the test page (i.e. the d.s. landing page). The value I changed does not affect whether or not it appeared on the test page itself. Rather, the value I changed affects whether or not it shows up on the library of templates and patterns.
